miner/worker.go

```diff
@@ -505,6 +578,14 @@ func (miner *Miner) commitTransactions(env *environment, plainTxs, blobTxs *tran
 			log.Warn("Skipping account, transaction with failed conditional", "sender", from, "hash", ltx.Hash, "err", err)
 			txs.Pop()
 
+		case env.rpcCtx != nil && env.rpcCtx.Err() != nil && errors.Is(err, env.rpcCtx.Err()):
+			log.Warn("Transaction processing aborted due to RPC context error", "err", err)
+			return errBlockInterruptedByTimeout // RPC timeout. Tx could not be checked.
```

Comment:
This should `continue` rather than error? this seems like it will cause the full block to be rebuilt from scratch

Reply:
Good catch